authors/apps/authentication/serializers.py

```diff
@@ -133,7 +140,7 @@ def validate(self, data):
         return {
             'email': user.email,
             'username': user.username,
-
```

Comment:
Why are we making a new line here

Reply:
I actually just deleted the line for neatness

Reply:
@ssewilliam, the line was just removed and not added.